WookieeMatt/Essence20
# Transitioning away from Handler classes

Addresses https://github.com/WookieeMatt/Essence20/issues/617

##### In this PR
- This is the first step in the process to split up `actor-sheet.mjs`. Initially these "handlers" were made so that they could store actors and actor sheets so that they wouldn't be needed as parameters, but now the file has grown large enough that it's becoming detrimental.
- This PR begins refactoring the handlers away from classes into standalone helper methods. It only starts with about half of them to limit the size of the PR and the amount of testing needed at once. Looking back, I probably should have started with fewer, but oh well.
- Removed unused methods `_populateChildItems` and `_addChildItemToParent`
- I cleaned up some comment blocks as well

##### Testing
- AdvancementHandler 
  - Changing levels
- AlterationHandler
   - Add and delete alterations on an actor
- BackgroundHandler 
  - Add and delete origins, influences, hangups on an actor
- CrossoverHandler 
  - Crossover options should work on an actor sheet
- PowerRangerHandler 
  - Morph button should work
  - Add and delete zords on a MFZ
- AttachmentHandler 
  - Drop a weapon from the compendium
  - Drop a WE on an actor with multiple weapons
